lib/link_thumbnailer/scrapers/opengraph/image.rb

```diff
@@ -20,7 +21,11 @@ def value
           end
 
           def model
-            nodes.map { |n| modelize(n, n.attributes['content'].to_s) }
+            nodes.map do |n|
+              if LinkThumbnailer::URI.new(n.attributes['content']).valid?
```

Comment:
Actually something like this should be enough and easier to reason about, what do you think?

```ruby
nodes.map do |n|
  uri = LinkThumbnailer::URI.new(n.attributes['content'])
  modelize(n, uri.to_s) if uri.valid?
end.compact
```